pom.xml

```diff
@@ -22,38 +22,62 @@
 
   <properties>
     <apachecommons.version>3.1</apachecommons.version>
+    <checkstyle.version>8.29</checkstyle.version>
     <commonsio.version>2.5</commonsio.version>
     <fastutil.version>8.1.1</fastutil.version>
+    <gitcommitplugin.version>2.2.6</gitcommitplugin.version>
     <glassfish.version>1.1</glassfish.version>
     <googlejson.version>1.1.1</googlejson.version>
     <guava.version>23.6-jre</guava.version>
     <h2.version>1.4.197</h2.version>
     <ignite.version>2.7.5</ignite.version>
+    <jacoco.version>0.8.5</jacoco.version>
     <jcommander.version>1.72</jcommander.version>
     <jetbrainsannotations.version>13.0</jetbrainsannotations.version>
+    <jts.version>1.16.1</jts.version>
+    <junit.version>4.12</junit.version>
     <kafka.version>0.11.0.0</kafka.version>
     <lz4.version>1.4.0</lz4.version>
     <mapdb.version>3.0.5</mapdb.version>
+    <mavenassembly.version>3.1.1</mavenassembly.version>
+    <mavencheckstyle.version>3.1.0</mavencheckstyle.version>
+    <mavenpmd.version>3.11.0</mavenpmd.version>
+    <mavenprojectinfo.version>3.0.0</mavenprojectinfo.version>
+    <mavensurefire.version>2.22.2</mavensurefire.version>
     <orgjson.version>20160810</orgjson.version>
     <osmosis.version>0.45</osmosis.version>
+    <pmd.version>6.11.0</pmd.version>
     <postgresql.version>42.1.4</postgresql.version>
     <protobuf.version>2.6.1</protobuf.version>
     <roaringbitmap.version>0.6.59</roaringbitmap.version>
+    <roaringbitmap.version>0.6.59</roaringbitmap.version>
```

Comment:
what's the change in this line?

Reply:
I just sorted the properties alphabetically. Now duplicates like this one are more noticeable :sweat_smile: 

I will remove this duplicate.